File: latihan1/Latihan B/C++/bola.cpp
#include <string>
#include <iostream>

using namespace std;

class bola{
    private:
        // atribut
        std::string nama, negara, pemain;
        int tahun;

    public:
        // constructor
        bola(){
            this->tahun = 0;
            this->nama = this->negara = this->pemain = "blank";
        }

        // constructor + parameter
        bola(string nama, string negara, int tahun, string pemain){
            this->nama = nama;
            this->negara = negara;
            this->tahun = tahun;
            this->pemain = pemain;
        }

        /*
            Setter dan Getter
        */ 

        void setNama(string nama){ //ini fungsinya cmn 1 atribut
			this->nama = nama;
		}
        string getNama(){
            return this->nama;
        }

        void setNegara(string negara){ //ini fungsinya cmn 1 atribut
			this->negara = negara;
		}
        string getNegara(){
            return this->negara;
        }

        void setTahun(int tahun){ //ini fungsinya cmn 1 atribut
			this->tahun = tahun;
		}
        int getTahun(){
            return this->tahun;
        }

        void setPemain(string pemain){ //ini fungsinya cmn 1 atribut
			this->pemain = pemain;
		}
        string getPemain(){
            return this->pemain;
        }

        // method menampilkan output
        void keluaran(){
            cout << "==============================================" << endl;
            cout << "Nama tim : " << this->nama << endl;
            cout << "Negara asal : " << this->negara << endl;
            cout << "Tahun berdiri : " << this->tahun << endl;
            cout << "Pemain : " << this->pemain << endl;
        }

        ~bola(){}

};

File: latihan1/Latihan B/Python/bola.py
class bola():
    # atribut
    __nama = "blank"
    __negara = "blank"
    __tahun = 0
    __pemain = "blank"

    # constructor
    def __init__(self, nama="blank", negara="blank", tahun=0, pemain="blank"):
        self.__nama = nama
        self.__negara = negara
        self.__tahun = tahun
        self.__pemain = pemain

    # Setter dan Getter

    def setNama(self, nama):
        self.__nama = nama
        
    def getNama(self):
        return self.__nama 

    def setNegara(self, negara):
        self.__negara = negara
        
    def getNegara(self):
        return self.__negara 

    def setTahun(self, tahun):
        self.__tahun = tahun
        
    def getTahun(self):
        return self.__tahun 

    def setPemain(self, pemain):
        self.__pemain = pemain
        
    def getPemain(self):
        return self.__pemain 

    # untuk menampilkan keluaran
    def keluaran(self):
        print ("==============================================")
        print ("Nama tim : %s" % (self.__nama)) 
        print ("Negara asal : %s" % (self.__negara)) 
        print ("Tahun berdiri : %d" % (self.__tahun)) 
        print ("Pemain : %s" % (self.__pemain)) 